### PR TITLE
Update Signer guide 

### PR DESCRIPTION
**Pull request by GloreenS**
- Added password requirements.
- Added connected sites
- Minor formatting changes

### Related links

// Paste links to Github tasks or links to related PRs.

### Changes

// Paste brief description what has changed.

// Paste [loom](https://www.loom.com/), screenshots, or gifs([record](https://giphy.com/apps/giphycapture), [compress](https://gifcompressor.com/)) here.

### Notes

// Paste any other notes if any

### CODE REVIEW

File: source/docs/casper/workflow/signer-guide.md [hunk 1]
```diff
@@ -97,18 +102,36 @@ This feature allows you to view all the websites and decentralized applications
...
+:::
 3.  Click the main menu icon <img src="/static/image/tutorials/signer/main-icon.png" width="25" class="inline-img" /> to open the main menu and select **Connected Sites**. The current website is displayed in the list of connected sites. 
 4.  To connect to a site, click the connect icon next to the name of the website.
 
 **Figure 4**: Connected Sites
 <img src="/static/image/tutorials/signer/connect-site.png" width="200" />
 
-**Note**: If the website is not integrated with the Casper Signer, the roll-over text displays the message ‘This site is not integrated with the Signer’.
+
+#### 4.3.2	List of websites than can connect to the Casper Signer
```
Comment (ipopescu) [inline source/docs/casper/workflow/signer-guide.md:120]: Use "that" instead of "than"

Comment (ipopescu) [inline source/docs/casper/workflow/signer-guide.md:120]: During the review, I've been wondering if we really need this section (4.3.2). This list is going to grow and I wonder if we should leave it here or take it out. I know the original task was to add it, but now I am doubting that.

**Review by ipopescu — APPROVED** [review]
Nice work, @GloreenS!
I left a few minor comments in the review.

File: source/docs/casper/workflow/signer-guide.md (hunk 1 above)
Comment (GloreenS) [reply source/docs/casper/workflow/signer-guide.md:120]: I agree, this is an incremental list, it already has a couple more websites added. Will remove the section. Thanks!

**Pull request merged**